rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
+);
+
+const create = context => {
+	return {
+		CallExpression(node) {
+			const callee = node.callee;
+			const args = node.arguments;
+
```

Comment:
It needs to take in account that it doesn't work if the user uses the `i` flag. For example: `/^bar/i.test(foo);`